tests/AppBundle/Repository/EventRepositoryTest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\AppBundle\Repository;
+
+use AppBundle\Repository\EventRepository;
+use Tests\AppBundle\EventMetricsTestCase;
+
+/**
+ * @covers EventRepository
+ */
+class EventRepositoryTest extends EventMetricsTestCase
+{
+    public function testGetPagesUsingFile():void
+    {
+        $kernel = static::bootKernel();
+        /** @var \Doctrine\ORM\EntityManager $entityManager */
+        $entityManager = $kernel->getContainer()->get('doctrine')->getManager();
+        $repo = new EventRepository($entityManager);
+        $repo->setContainer($kernel->getContainer());
+
+        static::assertGreaterThan(0, $repo->getPagesUsingFile('commonswiki_p', 'Ultrasonic_humidifier.jpg'));
+        static::assertGreaterThan(0, $repo->getPagesUsingFile('enwiki_p', '2-cube.png'));
+    }
+}
```

Comment:
We don't usually test repositories (comment for the EventRepository class is even marked with `@codeCoverageIgnore`), but I'm certainly not opposed to it :smiley: I think it'd be good to have some integration testing. The "extended" fixture includes local file uploads that should always have more than zero pages using the files: https://github.com/wikimedia/eventmetrics/blob/master/src/AppBundle/DataFixtures/ORM/extended.yml#L47-L51 So in ProcessEventCommandTest you should be able to add a `pagesWithFilesSpec()`, like the others, making assertions against the relevant EventStat.